book/src/languages.md

```diff
@@ -62,6 +62,7 @@ These configuration keys are available:
 | `grammar`             | The tree-sitter grammar to use (defaults to the value of `name`) |
 | `formatter`           | The formatter for the language, it will take precedence over the lsp when defined. The formatter must be able to take the original file as input from stdin and write the formatted file to stdout |
 | `max-line-length`     | Maximum line length. Used for the `:reflow` command and soft-wrapping           |
+| `workspace-roots`     | Directories relative to the workspace root that are treated as LSP roots. Should only be set in `.helix/config.toml`. Overwrites the setting of the same name in `config.toml` if set. | `` |
```

Comment:
`workspace-roots` (`Directories...`) means something else than `the workspace root` mentioned in the `Description` column.

Could this better be named `language-server-roots` / `lsp-roots`?

Reply:
Hmm not sure. The reason I called this workspace-roots is that this is supposed to be something that is configured per workspace (in `.helix/config.toml`) so this is meant as `roots` inside the workspace. Maybe `workspace-lsp-roots` works better 🤔

Reply:
Sounds perfect to me. `workspace-lsp-roots` transmits both the workspace-specificity as well as the lsp-target.